obal/data/modules/changelog.py

```diff
@@ -50,6 +50,9 @@ def main():
 
     changed = False
 
+    if not changelog.starts_with('-'):
```

Comment:
Looks like a Rubyism
```suggestion
    if not changelog.startswith('-'):
```